acmw-db-app/pages/signin.js

```diff
@@ -13,23 +18,31 @@ const SignIn = () => {
       <main className={styles.main}>
         <div className={styles.card}>
           <img src="/logo.png" alt="ACM-W Logo" className={styles.logo} />
-          <h1> OSU Database </h1>
+          <h1> ACM-W OSU Database </h1>
           <h2> Please sign in to continue </h2>
           <TextInput
             id="username"
-            label="Username"
-            predicted="Brutus Buckeye"
+            label="Email"
+            predicted="brutus@osu.edu"
```

Comment:
can we have `buckeye.1@osu.edu` i know it doesn't matter that much but :)

Reply:
done